content/en/platform/corda/5.1/reference/certificates.md

```diff
@@ -0,0 +1,84 @@
+| CPB Signing      | Code-signing key for publishing CPIs. The CorDapp publisher uses this key to sign a {{< tooltip >}}CPB{{< /tooltip >}} before sending it to the Network Operator to use when [creating the CPI]({{< relref "../application-networks/creating/members/cpi.md#import-code-signing-certificates" >}}). For more information, see [Code Signing]({{< relref "../developing-applications/packaging/code-signing/_index.md">}}).                                                                   | 256-bit X.509 public signature verification key |
+| CPI Publisher    | Code-signing key for publishing CPIs. Corda validates that uploaded {{< tooltip >}}CPIs{{< /tooltip >}} are signed with a trusted key. The Network Operator usually [imports the signing key to Corda]({{< relref "../application-networks/creating/members/cpi.md#import-code-signing-certificates" >}}). In some cases, such as the Notary CPI, R3 manages the key. For more information, see [Code Signing]({{< relref "../developing-applications/packaging/code-signing/_index.md">}}). | 256-bit X.509 public signature verification key | 256-bit X.509 public signature verification key
+
+## PKI assets
+
+Corda supports the following PKI assets:
+
+| PKI                            | Description                                                                                                                                                                                            | Type              |
+| ------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ----------------- |
+| Kafka Truststore               | Kafka truststore for server authentication (TLS).                                                                                                                                                      | JKS               |
+| Session Keys X.509 Certificate | Session keys certificate validated during session initialization. For more information, see [Session Certificates]({{< relref "../application-networks/creating/optional/session-certificates.md" >}}). | X.509 Certificate |
+| E2E Identity Trust Roots       | Trust root of certificates for session initiation keys.                                                                                                                                                | 256-bit PEM       |
+| Certificate for Corda releases | Trusted certificates for binary Corda release verification.                                                                                                                                            | X.509 Certificate |
+
+## Credentials
```

Comment:
Some credentials in this table do not have a type provided. I assumed it's on purpose.